packages/knex/src/query/QueryBuilderHelper.ts

```diff
@@ -20,12 +20,12 @@ import { JoinOptions } from '../typings';
 export class QueryBuilderHelper {
 
   constructor(private readonly entityName: string,
-              private readonly alias: string,
-              private readonly aliasMap: Dictionary<string>,
-              private readonly subQueries: Dictionary<string>,
-              private readonly metadata: MetadataStorage,
-              private readonly knex: Knex,
-              private readonly platform: Platform) { }
+    private readonly alias: string,
+    private readonly aliasMap: Dictionary<string>,
+    private readonly subQueries: Dictionary<string>,
+    private readonly metadata: MetadataStorage,
+    private readonly knex: Knex,
+    private readonly platform: Platform) { }
```

Comment:
lets remove unrelated code style changes

```suggestion
              private readonly alias: string,
              private readonly aliasMap: Dictionary<string>,
              private readonly subQueries: Dictionary<string>,
              private readonly metadata: MetadataStorage,
              private readonly knex: Knex,
              private readonly platform: Platform) { }
```